monster-cards/src/index.html

```diff
@@ -4,9 +4,6 @@
 		<meta charset="utf-8">
 		<title>dojo2 Monster Cards</title>
 	</head>
-	<body is="app-projector" data-css-transitions>
```

Comment:
turns out you don't (as this works) but I didn't actually mean to remove the `body`. An oversight on culling the `index.html`